src/scales/scale.time.js

```diff
@@ -477,6 +485,14 @@ module.exports = Scale.extend({
 			console.warn('options.time.format is deprecated and replaced by options.time.parser.');
 		}
 
+		if (time.min) {
```

Comment:
`time.min: 0` is a valid value, we should check for `time.min !== undefined`

Reply:
good catch